litgpt/__main__.py

```diff
@@ -1,5 +1,26 @@
 # Copyright Lightning AI. Licensed under the Apache License 2.0, see LICENSE file.
 
+import warnings
+import re
+# There is currently a PyTorch bug that raises a false-positive warning.
+# For more information, see https://github.com/Lightning-AI/litgpt/issues/1561
+warning_message = (
+    "The epoch parameter in `scheduler.step()` was not necessary and is being deprecated "
+    "where possible. Please use `scheduler.step()` to step the scheduler. During the deprecation, "
+    "if epoch is different from None, the closed form is used instead of the new chainable form, "
+    "where available. Please open an issue if you are unable to replicate your use case: "
+    "https://github.com/pytorch/pytorch/issues/new/choose."
+)
+
+
+def custom_filter(warning):
+    return issubclass(warning.category, UserWarning) and re.match(re.escape(warning_message), str(warning.message))
```

Comment:
Did you try without it? It shouldn't be necessary. The warning is a regular UserWarning

Reply:
Thanks, for some reason the warning still went through with a simpler approach but I am currently working on improving it. Thanks for the feedback here, I am not super familiar with filterwarnings!